File: 01_gpio_digital/LED_3.c
#include <wiringPi.h>
#define LED_RED 10
#define LED_YELLOW 9
#define LED_GREEN 11
int main (void)
{
  //wiringPiSetup () ;
  wiringPiSetupGpio();
  pinMode (LED_RED, OUTPUT) ;
  pinMode (LED_YELLOW, OUTPUT) ;
  pinMode (LED_GREEN, OUTPUT) ;
  for (int i=0; i<5; i++)
  {
    digitalWrite (LED_RED, HIGH) ; delay (1000) ;
    digitalWrite (LED_RED,  LOW) ; delay (1000) ;
    digitalWrite (LED_YELLOW, HIGH) ; delay (1000) ;
    digitalWrite (LED_YELLOW,  LOW) ; delay (1000) ;
    digitalWrite (LED_GREEN, HIGH) ; delay (1000) ;
    digitalWrite (LED_GREEN,  LOW) ; delay (1000) ;

  }
  return 0 ;
}

File: 01_gpio_digital/LED_3.py
import RPi.GPIO as GPIO
import time

LED_RED = 10
LED_YELLOW = 9
LED_GREEN = 11
GPIO.setmode(GPIO.BCM)   # GPIO.BCM or GPIO.BOARD
GPIO.setup(LED_RED, GPIO.OUT)  # GPIO.OUT or GPIO.IN
GPIO.setup(LED_YELLOW, GPIO.OUT)
GPIO.setup(LED_GREEN, GPIO.OUT)

for i in range(1):
    GPIO.output(LED_RED, GPIO.HIGH) # 1
    print("led RED on")
    time.sleep(2)
    GPIO.output(LED_RED, GPIO.LOW) # 0
    print("led RED off")
    time.sleep(2)
    GPIO.output(LED_YELLOW, GPIO.HIGH) # 1
    print("led YELLOW on")
    time.sleep(2)
    GPIO.output(LED_YELLOW, GPIO.LOW) # 0
    print("led YELLOW off")
    time.sleep(2)
    GPIO.output(LED_GREEN, GPIO.HIGH) # 1
    print("led GREEN on")
    time.sleep(2)
    GPIO.output(LED_GREEN, GPIO.LOW) # 0
    print("led GREEN off")
    time.sleep(2)

GPIO.cleanup() # GPIO 핀상태 초기화